<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TKNTumParcaListesi</class>
 <widget class="QDialog" name="TKNTumParcaListesi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Arama</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Parça Markası</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Değer</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="leParcaAdDegeri"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string>Filtre Türü</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QRadioButton" name="rbParcaAdiIleBaslayan">
                    <property name="text">
                     <string>İle Başlayan</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="rbParcaAdiIleBiten">
                    <property name="text">
                     <string>İle Biten</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="rbParcaAdiIceren">
                    <property name="text">
                     <string>İçeren</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Parça Mdeli</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Değer</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="leParcaModelDegeri"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_5">
               <property name="title">
                <string>Filtre Türü</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QRadioButton" name="rbParcaModelIleBaslayan">
                    <property name="text">
                     <string>İle Başlayan</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="rbParcaModelIleBiten">
                    <property name="text">
                     <string>İle Biten</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="rbParcaModelIceren">
                    <property name="text">
                     <string>İçeren</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="cbYazarkenAra">
          <property name="text">
           <string>Yazarken Ara</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Ara</string>
          </property>
          <property name="autoRepeatDelay">
           <number>300</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Kapat</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>TKNTumParcaListesi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>626</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leParcaAdDegeri</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TKNTumParcaListesi</receiver>
   <slot>Ara()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>646</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leParcaModelDegeri</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TKNTumParcaListesi</receiver>
   <slot>Ara()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>646</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbParcaModelIceren</sender>
   <signal>clicked()</signal>
   <receiver>TKNTumParcaListesi</receiver>
   <slot>Ara()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>822</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>646</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbParcaModelIleBiten</sender>
   <signal>clicked()</signal>
   <receiver>TKNTumParcaListesi</receiver>
   <slot>Ara()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>700</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbParcaModelIleBaslayan</sender>
   <signal>clicked()</signal>
   <receiver>TKNTumParcaListesi</receiver>
   <slot>Ara()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>579</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbParcaAdiIceren</sender>
   <signal>clicked()</signal>
   <receiver>TKNTumParcaListesi</receiver>
   <slot>Ara()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbParcaAdiIleBiten</sender>
   <signal>clicked()</signal>
   <receiver>TKNTumParcaListesi</receiver>
   <slot>Ara()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbParcaAdiIleBaslayan</sender>
   <signal>clicked()</signal>
   <receiver>TKNTumParcaListesi</receiver>
   <slot>Ara()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbYazarkenAra</sender>
   <signal>clicked(bool)</signal>
   <receiver>pushButton_2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>TKNTumParcaListesi</receiver>
   <slot>Ara()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>663</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>888</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Ara()</slot>
 </slots>
</ui>
